### PR TITLE
Fix links to API methods in the code splitting article

### PR DESCRIPTION
**Pull request by stof**
Fragments in the links were not matching the actual anchors.

### CODE REVIEW

File: src/content/guides/code-splitting.md [hunk 1]
```diff
@@ -162,7 +162,7 @@ Here are some other useful plugins and loaders provided by the community for spl
 
 ## Dynamic Imports
 
-Two similar techniques are supported by webpack when it comes to dynamic code splitting. The first and recommended approach is to use the [`import()` syntax](/api/module-methods#import-) that conforms to the [ECMAScript proposal](https://github.com/tc39/proposal-dynamic-import) for dynamic imports. The legacy, webpack-specific approach is to use [`require.ensure`](/api/module-methods#require-ensure). Let's try using the first of these two approaches...
+Two similar techniques are supported by webpack when it comes to dynamic code splitting. The first and recommended approach is to use the [`import()` syntax](/api/module-methods#import-1) that conforms to the [ECMAScript proposal](https://github.com/tc39/proposal-dynamic-import) for dynamic imports. The legacy, webpack-specific approach is to use [`require.ensure`](/api/module-methods#requireensure). Let's try using the first of these two approaches...
```
Comment (EugeneHlushko) [inline src/content/guides/code-splitting.md]: We have a case with dev-server when developing locally, not able to resolve links without trailing slash, lets fix that one here as well:

```suggestion
Two similar techniques are supported by webpack when it comes to dynamic code splitting. The first and recommended approach is to use the [`import()` syntax](/api/module-methods#import-1) that conforms to the [ECMAScript proposal](https://github.com/tc39/proposal-dynamic-import) for dynamic imports. The legacy, webpack-specific approach is to use [`require.ensure`](/api/module-methods/#requireensure). Let's try using the first of these two approaches...
```

**Comment by EugeneHlushko** [comment]
Please sign CLA

**Comment by EugeneHlushko** [comment]
Hi @stof 
Please sign CLA

**Comment by stof** [comment]
OK, I know why the CLA bot failed. As I made this edit through the github UI, the commit is made using the github-specific email address. But the bot does not seem to handle that case.

**Comment by stof** [comment]
hmm no, that's not it. the commit is with my email. Then it is a bug with the bot

**Comment by stof** [comment]
OK, forcing a push seems to have fixed it. So it looks like the bot did not update the status of the PR after I signed the CLA...

**Comment by stof** [comment]
OK, and now, Travis does not seem to run for the force push :disappointed:

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**